kex.c

```diff
@@ -62,6 +62,8 @@
 #include "sshbuf.h"
 #include "digest.h"
 
+#include "telemetry.h"
```

Comment:
Any code in common code (shared between windows and upstream) must be encapsulated with
#ifdef WINDOWS
#endif

Please it as a global comment and change in all the places